x/ccv/provider/keeper/keeper.go

```diff
@@ -722,9 +722,6 @@ func (k Keeper) DeleteValsetUpdateBlockHeight(ctx sdk.Context, valsetUpdateId ui
 }
 
 // SetSlashAcks sets the slash acks under the given chain ID
-//
-// TODO: SlashAcks should be persisted as a list of ConsumerConsAddr types, not strings.
-// See https://github.com/cosmos/interchain-security/issues/728
 func (k Keeper) SetSlashAcks(ctx sdk.Context, chainID string, acks []string) {
```

Comment:
See https://github.com/cosmos/interchain-security/issues/728#issuecomment-1620706001, we need to change the API but keep the serialized wire bytes the same
```suggestion
func (k Keeper) SetSlashAcks(ctx sdk.Context, chainID string, acks []types.ConsumerConsAddress) {
```